src/docs/testing/errors.md

```diff
@@ -52,6 +52,25 @@ This handler can also be used to report errors to a logging service.
 For more details, see our cookbook chapter for 
 [reporting errors to a service][].
 
+To customize error widget at root, set [`MaterialApp.builder`]:
```

Comment:
Hi @polina-c, nice to "meet" you and thanks for this PR! But I have some questions. When you say "to customize error widget at root", do you mean to set customize the error widget at the root of the widget tree so it is available to all widgets? Is something else going on? I need a bit more context.

Also, I'm not sure this bit belongs at the end of the page. If you want to talk offline, just drop me a message or an email..